collections/ansible_collections/purestorage/flashblade/plugins/modules/purefb_info.py

```diff
@@ -830,6 +831,10 @@ def generate_fs_dict(blade):
                 'is_local': fsys.items[fsystem].source.is_local,
                 'name': fsys.items[fsystem].source.name
             }
+        fs_info[share]['policies'] = []
```

Comment:
This added section needs to indent to be under the control of the version check.